src/sentry/notifications/helpers.py

```diff
@@ -715,6 +716,36 @@ def get_recipient_from_team_or_user(user_id: int | None, team_id: int | None) ->
     return recipient
 
 
+def team_is_valid_recipient(team: Team | RpcActor) -> bool:
+    from sentry.models.integrations.external_actor import ExternalActor
+
+    linked_integration = ExternalActor.objects.filter(
+        team_id=team.id, provider__in=TEAM_NOTIFICATION_PROVIDERS
```

Comment:
@isabellaenriquez can we just use `PERSONAL_NOTIFICATION_PROVIDERS`? There won't be rows for Discord/MSteams because linking isn't supported.

Reply:
ahh gotcha, will do

Reply:
Okay so the only issue is that ExternalActor expects an integer (value of `ExternalProvider`), but `PERSONAL_NOTIFICATION_PROVIDERS` is a list of `ExternalProviderEnum` values (ie. "slack"). I'm not sure there's a way to get the numerical equivalent from the string.